demos/utils.h

```diff
@@ -27,4 +28,26 @@ void FlushFromDataCache(const void *start, const void *end);
 #if SAFESIDE_LINUX
 void PinToTheFirstCore();
 #endif
+
+// Force a slow (i.e. uncached) memory access. Provides a "windowing gadget"[1]
+// that expands the speculative execution window by delaying the retirement of
```

Comment:
If the main purpose is extending the windowing, what about making that the subject of the function? i.e. name it `ExtendSpeculationWindow` and document that it delays retirement[0], and then document the mechanism second; as opposed to naming it `SlowMemoryAccess` and documenting that it performs a slow memory access, and then documenting that this delays retirement second.

Like I think it's clearer if we see:

```c++
complicated_code();
ExtendSpeculationWindow();
more_code();
```

whereas with the below, one might ask "what is the purpose of a slow memory access?":

```c++
complicated_code();
SlowMemoryAccess();
more_code();
```

([0]: Alternate name: "dot-com crash")

Reply:
Good call. Done.